ArjunaCore/arjuna/pom.xml

```diff
@@ -222,269 +222,273 @@
       </dependencies>
     </profile>
     <profile>
-        <id>release</id>
-        <build>
-            <plugins>
-              <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-install-plugin</artifactId>
-                <executions>
-                  <execution>
-                    <id>default-install</id>
-                    <phase>install</phase>
-                  </execution>
-                </executions>
-                <inherited>false</inherited>
-              </plugin>
-            </plugins>
-        </build>
+      <id>release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-install-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>default-install</id>
+                <phase>install</phase>
+              </execution>
+            </executions>
+            <inherited>false</inherited>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <!--
+    <profile>
+      <id>sybase-jdbc-store</id>
+      <dependencies>
+        <dependency>
+          <groupId>com.sybase.jConnect</groupId>
+          <artifactId>jconn3</artifactId>
+          <version>${version.com.sybase.jConnect}</version>
+          <scope>system</scope>
+          <systemPath>${orson.jar.location}/../qa/dbdrivers/jConnect-6_0/classes/jconn3.jar</systemPath>
+        </dependency>
+      </dependencies>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <systemProperties>
+                <property>
+                  <name>com.arjuna.ats.arjuna.common.propertiesFile</name>
+                  <value>sybasejbossts-properties.xml</value>
+                </property>
+              </systemProperties>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    -->
+    <profile>
+      <id>oracle-jdbc-store</id>
+      <dependencies>
+        <dependency>
+          <groupId>com.oracle.database.jdbc</groupId>
+          <artifactId>ojdbc8</artifactId>
+          <version>${version.com.oracle}</version>
+          <scope>test</scope>
+        </dependency>
+      </dependencies>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <systemProperties>
+                <property>
+                  <name>com.arjuna.ats.arjuna.common.propertiesFile</name>
+                  <value>oraclejbossts-properties.xml</value>
```

Comment:
would not be more conscise to define the property like `jbossts.propertyxml.file.name` and define it in `properties` section of the `profile` and setting up the than setting up the properties file just once in `build` parametrized with that properties?

Reply:
Sorry, this area was highlighted as "new" because of the modifications introduced by the checkstyle.

Having said that, I agree with you that the `pom.xml` can be improved with a property. @mmusgrov WDYT?

Reply:
okay

> Sorry, this area was highlighted as "new" because of the modifications introduced by the checkstyle.

This is a prime example of why reformatting code causes difficulties problem

Reply:
> This is a prime example of why reformatting code causes difficulties.

True. This is a good example. Fortunately, reformatting code only happens once :-) (and, according with what has been recently discussed for old files, it can take place when there is a functional modification)